examples/camera-controller/commands/clusters/ClusterCommand.h

```diff
@@ -81,9 +82,17 @@ class ClusterCommand : public InteractionModelCommands, public ModelCommand, pub
             return;
         }
 
-        if (data != nullptr)
+        if (data == nullptr)
         {
-            LogErrorOnFailure(RemoteDataModelLogger::LogCommandAsJSON(path, data));
+            return;
+        }
+
+        LogErrorOnFailure(RemoteDataModelLogger::LogCommandAsJSON(path, data));
+
+        // handle WebRTCTransportProvider responses explicitly
+        if (path.mClusterId == chip::app::Clusters::WebRTCTransportProvider::Id)
```

Comment:
This file is used to handle the auto generated cluster command, we should not hijack the OnResponse logic in this file, The WebRTCProvider Responses are handled in WebRTCProviderClient::OnResponse

Reply:
Thank you for your review, Since in YAML TC, command is invoked as a standalone cluster interaction without using custom commands like webrtc provide-offer 3, it is not sufficient if provide-offer-response is only handled via WebRTCProviderClient::OnResponse() The same handling is also required when the auto generated cluster command is used i.e. when provide-offer is invoked with webrtctransportprovider provide-offer WebRTCSessionID Sdp StreamUsage OriginatingEndpointID destination-id endpoint-id-ignored-for-group-commands